test/unit/spec/util/constants.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable new-cap */
+'use strict';
+
+const assert = require('assert');
+
+const Constants = require('../../../../lib/util/constants');
+
+describe('Constants', () => {
+  describe('WS_SERVER.', () => {
```

Comment:
@makarandp0 

IMHO, these tests can be moved to the `connect` unit tests:

*test/unit/spec/connect.js*
```js
describe('connect', () => {
  ...
  describe('called with ConnectOptions#region', () => {
    [/* Values for region */].forEach(region => {
      context(`When region is "${region}"`, () => {
        ...
      });
    });
  });

  describe('called with ConnectOptions#environment', () => {
    [/* Values for environment */].forEach(environment => {
      context(`When environment is "${environment}"`, () => {
        ...
      });
    });
  });
  ...
});
```

Reply:
I did add a test to connect to make sure that integration between connect and util works. But these unit tests, I think are more useful to be closer to implementation.

Reply:
now I have added similar tests to connct.js as well.

Reply:
@makarandp0 

Since `utils` is mostly used by other modules, we should test them by actually testing the context in which they will be used. That way, when anyone reads the tests, they know that ConnectOptions#region is expected to behave a certain way.

Reply:
@manjeshbhargav,  I did take your suggestion and added tests in connect.js My reasoning for keeping the tests in constants.WS_SERVER was the ability to go into details of how the format of the url  looks like for example:

```
    it('maps gll to global for region', () => {
      const wsServer = Constants.WS_SERVER('anyenvironment', 'gll');
      assert.equal(wsServer, 'wss://global.vss.anyenvironment.twilio.com/signaling');
    });
```
where as similar test when written from connect.js need to  be bit vague about the actual format as they do not need to know what Constants.WS_SERVER does interenally. So I had something like:
```
        if (region === 'gll') {
          assert(options.wsServer.indexOf('global') > 0);
        } else {
          assert(options.wsServer.indexOf(region) > 0);
        }
```